diff --git dist/.gitignore dist/.gitignore
index 6f291dc..d105986 100644
--- dist/.gitignore
+++ dist/.gitignore
@@ -38,8 +38,6 @@ Release_TS_inline
 Release_inline
 ZendEngine1
 _libs
-acconfig.h
-aclocal.m4
 autom4te.cache
 bsd_converted
 buildconf.stamp
@@ -59,16 +57,13 @@ configuration-parser.c
 configuration-parser.h
 configuration-parser.output
 configuration-scanner.c
-configure
 configure.bat
 configure.js
 conftest
 conftest.c
 debug.log
 diff
-generated_lists
 include
-install-sh
 internal_functions.c
 lcov_data
 lcov_html
@@ -76,8 +71,6 @@ libs
 libtool
 meta_cc
 meta_ccld
-missing
-mkinstalldirs
 modules
 php
 php-*.tar.gz
@@ -96,17 +89,10 @@ stamp-h.in
 scan_makefile_in.awk
 TSRM/tsrm_config.h
 Zend/zend_config.h
-Zend/zend_ini_parser.c
-Zend/zend_ini_parser.h
-Zend/zend_ini_parser.output
-Zend/zend_language_parser.c
-Zend/zend_language_parser.h
-Zend/zend_language_parser.output
 ext/date/lib/timelib_config.h
 ext/iconv/php_iconv_aliased_libiconv.h
 ext/phar/phar.php
 main/php_config.h
-main/php_config.h.in
 
 # Test specific Ignores
 tests/*/*.diff
