diff --git dist/main/main.c dist/main/main.c
index 4e0294c..1bff590 100644
--- dist/main/main.c
+++ dist/main/main.c
@@ -1102,8 +1102,6 @@ static void php_error_cb(int type, const char *error_filename, const uint error_
 		/* no break - intentionally */
 		case E_ERROR:
 		case E_RECOVERABLE_ERROR:
-		case E_PARSE:
-		case E_COMPILE_ERROR:
 		case E_USER_ERROR:
 			EG(exit_status) = 255;
 			if (module_initialized) {
